doc/en/new-docs/user/commandlineuseful.rst

```diff
@@ -5,16 +5,98 @@ Most useful command-line options
+
+  def test_fail():
+      print "this test is going to fail"
+      assert False
+
+
+  def test_pass():
+      print "this test is going to pass"
+      assert True
+
+What will be printed will in the terminal be the following::
+
+  test_documentation.py .F
+
+  =================================== FAILURES ===================================
+  __________________________________ test_fail ___________________________________
+
+      def test_fail():
+          print "this test is going to fail"
+  >       assert False
+  E       assert False
+
+  test_documentation.py:7: AssertionError
+  ----------------------------- Captured stdout call -----------------------------
+  this test is going to fail
+  ====================== 1 failed, 1 passed in 0.02 seconds ======================
+
+But if it is run with the parameter (``py.test test_documentation.py -s``), the result showed will be the following one::
+
+  ====================================================== test session starts ======================================================
+  test_documentation.py this test is going to fail
+  F this test is going to pass
+  .
+
+  =========================================================== FAILURES ============================================================
+  ___________________________________________________________ test_fail ___________________________________________________________
+
+      def test_fail():
+          print "this test is going to fail"
+  >       assert False
+  E       assert False
+
+  test_documentation.py:7: AssertionError
+  ============================================== 1 failed, 1 passed in 0.02 seconds ===============================================
+
+The biggest difference in the both statements is the part entitled "Tests session starts", where it prints the statements "this test is going to pass" (so the test that passed), and the "this test is going to fail".
+
+-k EXPRESSION
+-------------
+
+EXPRESSION is a key word to select a subset of tests to be run.
+
+An example of this using the same file as above could be the following one:
+if this file is run without the parameter (if you run in the terminal, ``py.test test_documentation.py``), both tests on the file would be run. If you run that with the parameter (typing, for example ``py.test test_documentation.py -k pass``), it would filter only the tests names that contains the expression 'pass', so it would run only the test_pass().
+
+-v, --verbose
+-------------
+It shows more details of the tests, for example the test names.
 
 --collect-only
+--------------
+Shows a list of the tests without running them. It can be useful in combination with ``-k``.
+
+for example, in the test file above, if it was run with the collect-only argument, it would display as a result something like the bellow file:
```

Comment:
Probably this should end with `::` and the output text itself be indented so it appears as a monospace font.